### ISSUE TITLE
[React Native] Error: Unable to resolve module

### ISSUE THREAD

**Issue by hellogerard** [VirgilSecurity/virgil-e3kit-js#57]
Hello,

Trying to use `0.6.0-beta.1` in a React Native app. I have all the underyling dependencies installed, but on app startup in the iOS simulator, I am getting the error below. 

```
Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `./dist/libfoundation.browser.wasm` from `/Users/gerard/_work/crossroads-anywhere/react-native/node_modules/@virgilsecurity/e3kit/browser.cjs.js`: The module `./dist/libfoundation.browser.wasm` could not be found from `/Users/gerard/_work/crossroads-anywhere/react-native/node_modules/@virgilsecurity/e3kit/browser.cjs.js`. Indeed, none of these files exist:
  * `/Users/gerard/_work/crossroads-anywhere/react-native/node_modules/@virgilsecurity/e3kit/dist/libfoundation.browser.wasm(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
  * `/Users/gerard/_work/crossroads-anywhere/react-native/node_modules/@virgilsecurity/e3kit/dist/libfoundation.browser.wasm/index(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
```

Just checking on disk, the `node_modules/@virgilsecurity/e3kit/dist/libfoundation.browser.wasm` is there.

Also, I'm on React Native 0.59. I know native package management changed a lot in 0.60, but that shouldn't matter here, should it?   Can anyone help?

**Comment by vadimavdeev** [comment]
Hello
It seems like you import the browser bundle instead of the react native one. How do you import `EThree`? It should be done like this (note the trailing `/native`):
```js
import { EThree } from '@virgilsecurity/e3kit/native';
```
This way react native bundler won't try to import any `.wasm` modules

**Comment by vadimavdeev** [comment]
With react-native@0.59 I also needed to manually link the [react-native-keychain](https://github.com/oblador/react-native-keychain) library and move the `Pods_<MyTargetName>.framework` up before all of the libraries in XCode -> Target -> General -> Linked Frameworks and Libraries section

**Comment by hellogerard** [comment]
Okay, I got it importing. I was doing the above things more or less. But my original import looked like this:

```js
import { EThree, WrongKeyknoxPasswordError, IdentityAlreadyExistsError } from '@virgilsecurity/e3kit';
```

I am importing the error types so that, for example, if the backup password is invalid, I can allow the user to try again, versus other types of errors. I noticed right away that these are not exported anymore, but I thought leaving them in wouldn't hurt the import per se. Removing these fixes the import error.

My next question would be.... is there any reason these errors were removed from the named exports? Could be add them back in? Might be nice for developers to be have access to these types.

**Comment by vadimavdeev** [comment]
Yep, that's a mistake, we didn't do it on purpose. We'll add them back. For now, you can check the `name` property of the error object, which corresponds to the class name. For example:

```
try {
  await e3.register();
} catch (err) {
  if (err.name === 'IdentityAlreadyExistsError') {
     doSomething();
  } else {
    throw err;
  }
}
```
 Personally, I prefer this method of checking the error type to `instanceof` checks
